application/src/main/data/upgrade/3.6.4/schema_update.sql

```diff
@@ -134,3 +134,33 @@ DELETE FROM asset WHERE type='TbServiceQueue';
 DELETE FROM asset_profile WHERE name ='TbServiceQueue';
 
 -- QUEUE STATS UPDATE END
+-- TENANT PROFILE UPDATE START
+
+UPDATE tenant_profile tp
+SET profile_data =
+        jsonb_set(
+                jsonb_set(
+                        jsonb_set(
+                                profile_data,
+                                '{configuration,transportGatewayMsgRateLimit}',
+                                profile_data->'configuration'->'transportDeviceMsgRateLimit'
+                        ),
+                        '{configuration,transportGatewayTelemetryMsgRateLimit}',
+                        profile_data->'configuration'->'transportDeviceTelemetryMsgRateLimit'
+                ),
+                '{configuration,transportGatewayTelemetryDataPointsRateLimit}',
+                profile_data->'configuration'->'transportDeviceTelemetryDataPointsRateLimit'
+        )
+WHERE jsonb_typeof(tp.profile_data #> '{configuration,transportGatewayMsgRateLimit}') IS NULL;
+
+-- TENANT PROFILE UPDATE END
+
+-- NOTIFICATION RULE UPDATE START
+UPDATE notification_rule
+SET trigger_config =
+        CASE
+            WHEN trigger_config::jsonb->'apis' ? 'TRANSPORT_MESSAGES_PER_GATEWAY' THEN trigger_config
+            ELSE jsonb_set(trigger_config::jsonb, '{apis}', (trigger_config::jsonb->'apis') || '"TRANSPORT_MESSAGES_PER_GATEWAY"', false)::text
```

Comment:
Please check how does notification look like (with default notification rule) when per-gateway rate limits are exceeded

Reply:
Like regular rate limits exceeded notification, just with different content - "transport messages per gateway" instead of "transport messages per device"